### PR TITLE
Partial support for passing keyword arguments to CallNode

### PR DESCRIPTION
**Pull request by alpaca-tc**
Support KeywordHashNode and resolving basic keyword argument calls.

I have confirmed that it works in a simple case, but it is probably still incomplete.
However, it will be better than the current behavior where an exception is raised by a call to a keyword arguments.

### CODE REVIEW

File: lib/typeprof/core/ast/value.rb [hunk 1]
```diff
@@ -287,6 +287,47 @@ def install0(genv)
       end
     end
 
+    class KeywordHashNode < Node
```
Comment (alpaca-tc) [inline lib/typeprof/core/ast/value.rb:290]: This is the same behavior as `HashNode`, should I remove this and use `HashNode` instead of?

**Comment by mame** [comment]
This is great! I was doing the same work, but you beated me.

File: lib/typeprof/core/ast/value.rb (hunk 1 above)
Comment (mame) [reply lib/typeprof/core/ast/value.rb:290]: It's the exact same definition, right? Then, let's reuse HashNode for now. We can separate it if needed later

**Review by mame — APPROVED** [review]

**Comment by mame** [comment]
Thank you!

**Pull request merged**